pkgpanda/__init__.py

```diff
@@ -403,7 +413,12 @@ def add(self, fetcher, id, warn_added=True):
 
         # Cleanup artifacts (if any) laying around from previous partial
         # package extractions.
-        check_call(['rm', '-rf', tmp_path])
+        if is_windows:
+            check_call(["powershell.exe", "-command",
+                        "& { get-childitem -erroraction silentlycontinue -path " + tmp_path +
+                        " | remove-item -recurse -force }"])
+        else:
+            check_call(['rm', '-rf', tmp_path])
```

Comment:
This is another stanza that gets duplicated throughout this PR. We should abstract this into a separate function, or replace it with something from the `os` module.